### PR TITLE
MNT Convert README.md to README.rst

### LINKED ISSUE #884: Convert the README.md to an .rst file
Our README.md is a markdown file, but we can have more flexibility if it's converted to an RST file, which would also be more consistent with the rest of the documentation.

The converted file can be checked for errors using `twine check`.

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #884

### CODE REVIEW

**Review by romanlutz — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+Reporting security issues
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To report security issues please send an email to
+``fairlearn-internal@python.org``.
+
+.. |Build Status| image:: https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=main
+   :target: https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=main
+.. |MIT license| image:: https://img.shields.io/badge/License-MIT-blue.svg
+.. |PyPI| image:: https://img.shields.io/pypi/v/fairlearn?color=blue
```
[the remark this answers is not in the document]
Comment (romanlutz) [reply README.rst:165]: Actually this PR looks ready, so if another @fairlearn/fairlearn-maintainers feels like merging it I'll make the adjustments in my PR.

[the remark this answers is not in the document]
Comment (romanlutz) [reply README.rst:165]: Since @adrinjalali opened this it probably makes sense if he merges it (assuming there are no issues I missed)

**Comment by romanlutz** [comment]
@Patil2099 can you delete the existing README.md in this PR as well?

**Comment by adrinjalali** [comment]
Could you please remove the original file as well? You can do so by `git rm README.md`, and then commit and push as usual.

**Comment by romanlutz** [comment]
I quickly pulled in the Discord changes from #891 (which I just merged). 
@Patil2099 addressed your concern @adrinjalali, so please take another look 🙂

**Review by adrinjalali — APPROVED** [review]
Thanks @Patil2099

**Pull request merged**